LICENSE

```diff
@@ -112,5 +112,5 @@ Affirmer's express Statement of Purpose.
   party to this document and has no duty or obligation with respect to this
   CC0 or use of the Work.
 
-For more information, please see
+For more information, please see.
```

Comment:
Not sure, but why should there be a dot at the end?

Reply:
because it's beautiful for no reason :))